app/javascript/tests/history/HistoryList.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { shallow } from 'enzyme';
+import { Button, Col, Collapse, Row } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import _ from 'lodash';
+import HistoryList from '../../components/history/HistoryList.js'
+import { mockPatient1} from '../mocks/mockPatients'
+import { mockHistory1, mockHistory2 } from '../mocks/mockHistories'
+import { nameFormatter, formatDate } from '../util.js'
+
+const goToMock = jest.fn();
+const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==';
+
+beforeEach(() => {
+  window.BASE_PATH = ""
+})
+
+describe('HistoryList', () => {
+  it('Properly renders all history components', () => {
+    const wrapper = shallow(<HistoryList patient_id={mockPatient1.id} histories={[]} authenticity_token={authyToken} 
+      history_types={{enrollment: "Enrollment", comment: "Comment"}}/>);
```

Comment:
This test is missing a lot of key elements on the history page.  I would pass in `[mockHistory1, mockHistory2` and test that there are two history components. In addition, be sure to test that every dropdown, button, input, tooltips, child components etc are all there

Reply:
added here: https://github.com/SaraAlert/SaraAlert/pull/793/commits/336f7f2565ad7b9798767f02dd44c3442b64135d